### ISSUE TITLE
View releases for a repo

### ISSUE THREAD

**Issue by CrixuAMG** [CodeHubApp/CodeHub#491]
As far as I van see, there is no way to view all releases for the current repo. Could this feature be added into the app?

**Comment by Arc676** [comment]
I agree that it would be nice to see the descriptions of each release from within the app. However, it's worth noting that you *can* view the source code at any given *tag* in the repository by tapping the branch selection and switching to the "tags" tab.
